vajram/src/main/java/com/flipkart/krystal/vajram/VajramDef.java

```diff
@@ -1,10 +1,13 @@
 package com.flipkart.krystal.vajram;
 
+import java.lang.annotation.ElementType;
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
 
 @Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
 public @interface VajramDef {
 
-  String value();
+  String value() default "";
```

Comment:
Should we have this value() method? Kind of creates a loophole for developers to provide Vajram class and Vajram name differently bypassing the Vajram naming guidelines. Also, the name provided can conflict with the existing Vajram names on the class path. If removed, it will ensure the developers are naming the Vajram correctly.

Reply:
This is a very good point.
Removed this in the latest commit.